app/routes/PairingProcess/utils/ethOperations.js

```diff
@@ -13,12 +13,13 @@ export const createAccountFromMnemonic = (mnemonic) => {
 
 export const generatePairingCodeContent = (privateKey) => {
   const startDate = new Date()
-  const expiryDate = new Date(startDate.setMinutes(startDate.getMinutes() + 10))
+  const expirationDate = new Date(startDate.setMinutes(startDate.getMinutes() + 10))
```

Comment:
I see this strange. Maybe something like:
```const expirationDate = new Date().setMinutes(startDate.getMinutes() + 10)```

Reply:
Your proposal is not the same as what is written in the code. The former returns milliseconds while the later returns a Date object.